src/pages/rewards/LaunchIncentivesPanel.tsx

```diff
@@ -73,7 +73,9 @@ const EstimatedRewards = () => {
     queryKey: `launch_incentives_rewards_${dydxAddress ?? ''}`,
```

Comment:
Can we add the `SEASON_ NUMBER` as part of the queryKey?

Maybe even pull out this query into a separate hook.